good-enough-practices-for-scientific-computing.tex

```diff
@@ -176,102 +172,55 @@ \section{Data Management}\label{sec:data}
   treatment variable from \texttt{1} vs.  \texttt{2} to
   \texttt{untreated} vs. \texttt{treated}, and replace artificial
   codes for missing data, such as ``-99'', with \texttt{NA}s,
-  a shorthand code for ``Not Available'' that is used as a special variable
-  in many programming languages \cite{white2013}.
+  a code used in most programming languages to indicate that data is ``Not Available''
+  \cite{white2013}.
 
-  \emph{Filenames}: Store
-  especially useful metadata as part of the filename itself, while
-  keeping the filename regular enough for easy pattern matching. For
-  example, a filename like \texttt{2016-05-alaska-b.csv} makes it easy
-  for both people and programs to select by year or by location.
+  \emph{Filenames}: Store especially useful metadata as part of the filename itself, while
+  keeping the filename regular enough for easy pattern matching. For example, a filename
+  like \texttt{2016-05-alaska-b.csv} makes it easy for both people and programs to select
+  by year or by location.
 
 \item
-  \recommend{Create analysis-friendly data}. Re-formatting data will
-  eliminate processing steps between loading the
-  data set and doing analyses.
-
-  \emph{Split columns} that contain more
-  than one variable's worth of information. For example, most analytical environments will read the value ``3.4 kg''
-  as character data, rather than numeric. It should be split into two columns (the mass ``3.4'' and
-  the units ``kg''), or record the units in the variable
-  name and/or metadata.
-
-  \emph{Combine columns} that only contain one variable's worth of
-  information when taken together. This is
-  characteristic of data that has been laid out for human eyeballs or
-  for manual data entry.  For example, there might be one row per
-  field site and then columns for measurements made at each of several
-  time points. It is convenient to store this in a ``short and wide''
-  form for data entry and inspection, but for most analyses it will be
-  advantageous to gather these columns into a variable of
-  measurements, plus an additional column that indicates the
-  time point.
-
-  \emph{Reformat values} to match your environment's built-in
-  parsing rules. For example, use a date-time format that will be
-  recognized automatically, or better yet, split dates into three
-  columns (year, month, and day) for easier sorting and filtering.
+  \recommend{Create analysis-friendly data}: Analysis can be much easier if you are
+  working with so-called ``tidy'' data \cite{wickham2014}. Two key principles:
```

Comment:
I made this dramatically shorter but kept the two main principles. People can read the cited work for more detail.